test/e2e/validation_test.go

```diff
@@ -450,10 +450,14 @@ func testCSRApproval(t *testing.T) {
 		}
 	}
 
-	// Scale the Cluster Machine Approver deployment back to 1.
-	expectedPodCount := int32(1)
+	// Re-enable the Cluster Machine Approver by deleting the config then kicking off a re-creation of the CMA deployment
+	err = testCtx.client.K8s.CoreV1().ConfigMaps(CMANamespace).Delete(context.TODO(), CMAConfigName, meta.DeleteOptions{})
+	require.NoErrorf(t, err, "unable to delete ConfigMap %s/%s", CMANamespace, CMAConfigName)
+
+	expectedPodCount := int32(0)
```

Comment:
It is plausible that CVO doesn't reset the replicas in time or there is bug in CVO that doesn't allow it to take effect. It would be safer to scale the deployment down and scale it back up.